### ISSUE TITLE
Samples compatible with Arduino-CmdMessenger

### ISSUE THREAD

**Issue by thijse** [harmsm/PyCmdMessenger#20]
Hi,

It would be great if you could provide samples that are compatible with those in [@https://github.com/thijse/Arduino-CmdMessenger/tree/master/examples](url)

**Comment by harmsm** [comment]
Sounds like a good idea.  Just to make sure, you're suggesting I write python code that replicates those examples so people can learn the libraries.  Or is there python code that is specifically incompatible with those examples?

**Comment by thijse** [comment]
Yes, I think it would be very nice and useful to have Python samples that more or less replicate the functionality of the C# samples, in the sense that they communicate with the Arduino samples. I think that has the potential to help many people to set up Arduino-Python communications.

**Comment by jhagberg** [comment]
Yes it would be super!
